### PR TITLE
doc/dpdk: add IPS setup docs for DPDK mode 

### PR DESCRIPTION
**Pull request by lukashino**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [X] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [X] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [X] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/issues/5511) ticket:

Follow-up of #7867 

Describe changes:
- grammar fixes

### CODE REVIEW

**Review by jufajardini — COMMENTED** [review]
A few inline comments :)

File: doc/userguide/setting-up-ipsinline-for-linux.rst [hunk 1]
```diff
@@ -276,3 +278,69 @@ and eBPF load balancing looks like the following: ::
...
+~~~~~~~~~~~~~~~~~~
+
+In the same way, as you would configure AF_PACKET IPS mode you can configure the DPDK capture module.
+Prior to starting with IPS (inline) setup, it is recommended to go over :ref:`dpdk-capture-module` manual page
+to understand the setup essentials.
+
+DPDK IPS mode, similarly to AF-Packet, uses two interfaces. Packets received on the first network interface
+(``0000:3b:00.1``) are transmitted over the second network interface (``0000:3b:00.0``) and similarly,
+packets received on the second interface (``0000:3b:00.0``) are transmitted
+on the first interface (``0000:3b:00.1``). Packets are not altered in any way in this mode.
```
[the remark this answers is not in the document]
Comment (lukashino) [reply doc/userguide/setting-up-ipsinline-for-linux.rst:292]: What would you say about "transmitted by the first interface"? To me "transmitted to" sounds like you are transmitting packets towards that interface but actually, you are using that interface to transmit packets out of it. Not sure about grammar tho

File: doc/userguide/setting-up-ipsinline-for-linux.rst [hunk 2]
```diff
@@ -276,3 +278,69 @@ and eBPF load balancing looks like the following: ::
 
 The eBPF file ``/usr/libexec/suricata/ebpf/lb.bpf`` may not be present on disk.
 See :ref:`ebpf-xdp` for more information.
+
+DPDK IPS mode
+~~~~~~~~~~~~~~~~~~
+
+In the same way, as you would configure AF_PACKET IPS mode you can configure the DPDK capture module.
```
[the remark this answers is not in the document]
Comment (lukashino) [reply doc/userguide/setting-up-ipsinline-for-linux.rst:285]: done

File: doc/userguide/setting-up-ipsinline-for-linux.rst [hunk 3]
```diff
@@ -276,3 +278,69 @@ and eBPF load balancing looks like the following: ::
...
+
+DPDK capture module can be configured in two inline modes:
+
+- TAP mode - creates a bridge between two NIC ports but does not drop packets
+- IPS mode - the same as TAP but packets can be dropped based on rules
+
+When using multiple interfaces, it is crucial to specify the number of threads in each interface.
+Additionally, it is required that the paired interfaces have the same amount of threads configured.
+DPDK capture module also requires having CPU affinity set in the configuration file.
+The following snippet is from the ``threading`` configuration section  ::
```
[the remark this answers is not in the document]
Comment (lukashino) [reply doc/userguide/setting-up-ipsinline-for-linux.rst:337]: I've added a reference to the threading section and I do not think putting something about DPDK in the section you mentioned would be beneficial. Neither AFP capture module has anything mentioned there, and DPDK shares a lot of behavior with AFP.

**Comment by lukashino** [comment]
Follow up #7876

File: doc/userguide/setting-up-ipsinline-for-linux.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (jufajardini) [reply doc/userguide/setting-up-ipsinline-for-linux.rst:292]: AH! Ok, sorry, I got it wrongly. Yes, your suggestion, of course. Definitely explains what happens better.

**Pull request closed without merging**